services/movie_session.py

```diff
@@ -42,3 +42,14 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    seats = []
+    for ticket in (
+            MovieSession.objects.get(
+            id=movie_session_id
+            ).ticket_set.all()):
+        seats.append({"row": ticket.row,
+                      "seat": ticket.seat})
```

Comment:
Simplify your code, and use filtering by Ticket.objects.filter() instead of MovieSession